tests/testthat/curly-curly/mixed-out.R

```diff
@@ -59,11 +59,14 @@ call({{ x }})
 
 ##  ............................................................................
 ##  multiple                                                                ####
-call({
-  1
-}, a + b, {
-  33 / f(c)
-})
+call(
+  {
+    1
+  }, a + b,
```

Comment:
I think the `a + b` needs to go on a separate line. Is this related to this pull request at all?

Rationale: Multiline expressions in function calls don't contain code from other arguments.

This may be missing from the style guide, but seems obvious to me: If an argument is so complex that it needs multiple lines, we don't want it interspersed with other arguments.

Reply:
In the above example, I agree. However, when we have

```r
call(
  {
    x + y
  }, {
    f(x)
    call(x = 2, f = max(n))
  }
)
```

I'd leave it as is opposed to 
```r
call(
  {
    x + y
  }, 
  {
    f(x)
    call(x = 2, f = max(n))
  }
)
```

Because indention helps seeing what belongs together and `}, {` for me just reads as a separator of two expressions (i.e. I don't read them as two braces anymore). What do you think.

Reply:
Can we enforce the second option in `strict` mode but leave the first in `permissive` mode?

Reply:
I'd leave the first case as is, but I would not change the second into the first. So if your code has `}, \n{`, styler should not change it to `}, {`, and vice versa. Whenever there is brace + comma + not a brace, I agree to break lines in any case. The `strict` argument is already too convoluted I think. If your suggestion gets accepted in tidyverse/style, I am happy to also implement it for `strict = TRUE`, but I think it's a detail anyways.

Reply:
Works for me. Thanks for looking into it!

Reply:
Ok no this is more complicated to implement. I'll stick to your rule.